examples/chip-tool/commands/common/CHIPCommand.cpp

```diff
@@ -85,6 +88,26 @@ CHIP_ERROR GetAttestationTrustStore(const char * paaTrustStorePath, const chip::
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR GetRevocationSet(const char * revocationSetPath, const chip::Credentials::RevocationSet ** revocationSet)
```

Comment:
Please split all work to add revocation set support to chip-tool to a follow-up/separate PR. This will allow us to get agreement on the interface without concerns over any implementation details.

Reply:
Addressed in latest commit.